### PR TITLE
Calaveras && add new 1559 checks && removed some old 1559 testnet specs

### PR DESCRIPTION
**Pull request by MarekM25**
Review in first: https://github.com/NethermindEth/nethermind/pull/3092

The most important thing for reviewers: please verify two new checks:
1. In TxValidator I added: assert transaction.max_fee_per_gas >= transaction.max_priority_fee_per_gas
2. In TxPoolTxSource (when we're building blocks) and in TransactionProcessor I added assert signer.balance >= transaction.gas_limit * transaction.max_fee_per_gas

### CODE REVIEW

File: src/Nethermind/Nethermind.Runner/Data/static-nodes.json [hunk 1]
```diff
@@ -1,2 +1,7 @@
 [
+	"enode://9e1096aa59862a6f164994cb5cb16f5124d6c992cdbf4535ff7dea43ea1512afe5448dca9df1b7ab0726129603f1a3336b631e4d7a1a44c94daddd03241587f9@3.9.20.133:30303",
+	"enode://8e6788508278e514268307f27528ffabd6c86e755b0abd5ef170502b6444b35231dd0c7183e902c6a566c650104fdcfa58c6de28b54533f3dc243af444a60f51@quilt.link:30308",
+	"enode://14bed0619ef778342cb1ed18702004cb29f4324b7d411fc3d20adc8fe2d15c7bb30c663bd8767eaa32eaadcebc227a24e21c006028e12b9c38b25ea5e3147316@217.182.200.86:30303",
+	"enode://39eb08bbfad87481553c471a63ff2a4b4885fffa4ff50f1cf46744d9ad6e2f764ede146fe4df563fa9ccda1a46b9b1a88fb08135e1bf1d71b320912499da773d@3.21.156.138:30303",
+	"enode://aec88fd902744bf67705c098bf532b01017ccc3a156395508e2d9c4e7c22699ecccae1e7316614f8a2d4c5698a9be3fe6151ee25b9ed4aa052f88e112c65387a@164.90.171.157:31558"
```
Comment (LukaszRozmej) [inline src/Nethermind/Nethermind.Runner/Data/static-nodes.json:6]: I think we don't want to push it here?

File: src/Nethermind/Nethermind.Runner/Properties/launchSettings.json [hunk 2]
```diff
@@ -86,6 +86,10 @@
     "Baikal": {
       "commandName": "Project",
       "commandLineArgs": "--config configs\\baikal.cfg --Init.DiagnosticMode MemDb"
+    },
```
Comment (LukaszRozmej) [inline src/Nethermind/Nethermind.Runner/Properties/launchSettings.json:89]: Do we want to remove Aulet from launchSettings?

**Review by LukaszRozmej — APPROVED** [review]

File: src/Nethermind/Nethermind.Runner/Data/static-nodes.json (hunk 1 above)
Comment (MarekM25) [reply src/Nethermind/Nethermind.Runner/Data/static-nodes.json]: Done, thx

File: src/Nethermind/Nethermind.Runner/Properties/launchSettings.json (hunk 2 above)
Comment (MarekM25) [reply src/Nethermind/Nethermind.Runner/Properties/launchSettings.json]: Done, thx

**Review by marcindsobczak — APPROVED** [review]

**Review by kristofgazso — APPROVED** [review]

**Pull request merged**